chainer/utils/experimental.py

```diff
@@ -0,0 +1,40 @@
+import chainer
+import inspect
+import warnings
+
+
+def experimental(api_name=None):
+    """A function for marking APIs as experimental
+
+    Developer of the API can mark it as *experimental* by calling
```

Comment:
Wording:
I suggest "The developer of an API can ..." instead of "Developer of the API can ...".